Apps/Src/BlinkLights.c

```diff
@@ -13,9 +13,10 @@ void Task_BlinkLight(void* p_arg){
     while(1){
         Lights_Set(LEFT_BLINK, (State) (Lights_Toggle_Read(LEFT_BLINK) ^ Lights_Read(LEFT_BLINK)));
         Lights_Set(RIGHT_BLINK, (State) (Lights_Toggle_Read(RIGHT_BLINK) ^ Lights_Read(RIGHT_BLINK)));
-        Display_SetLight(LEFT_BLINK, (State) (Lights_Toggle_Read(LEFT_BLINK) ^ Lights_Read(LEFT_BLINK)));
-        Display_SetLight(RIGHT_BLINK, (State) (Lights_Toggle_Read(RIGHT_BLINK) ^ Lights_Read(RIGHT_BLINK)));
         
+        // Maybe add a check for if this has changed so we're not sending this command all the time?
```

Comment:
A check here is probably good practice, since we don't actually have to change the light state if the blinkers aren't on.